### PR TITLE
Pdj 13 base style

### PR DESCRIPTION
**Pull request by bachhunght**
Hi @febdao Please check it

### CODE REVIEW

**Comment by tannguyen04** [comment]
Multidev  created successfully! [http://pr-9-pdj.pantheonsite.io/](http://pr-9-pdj.pantheonsite.io/)

File: web/wp-content/themes/pdjtheme/dist/scss/base/_btn.scss [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+
+  &--darkblue {
+    background-color: $darkblue;
+  }
+
+  &--green {
+    background-color: $green;
+  }
+
   &,
```
Comment (febdao) [inline web/wp-content/themes/pdjtheme/dist/scss/base/_btn.scss]: Please remove this line, it's unnecessary

File: web/wp-content/themes/pdjtheme/dist/scss/base/_btn.scss [hunk 2]
```diff
@@ -1,12 +1,34 @@
...
   &:focus,
   &:visited {
+    color: $white;
   }
 
   &:active,
   &:hover {
+    color: $white;
```
Comment (febdao) [inline web/wp-content/themes/pdjtheme/dist/scss/base/_btn.scss]: Don't need this code

File: web/wp-content/themes/pdjtheme/dist/scss/base/_typography.scss [hunk 3]
```diff
@@ -46,48 +46,41 @@ h6 {
...
+  cursor: pointer;
 
   &:focus,
   &:visited {
-    color: $primary-color;
-  }
-
-  &:hover {
-    color: $primary-color;
   }
```
Comment (febdao) [inline web/wp-content/themes/pdjtheme/dist/scss/base/_typography.scss]: Remove this block if it was empty

File: web/wp-content/themes/pdjtheme/dist/scss/layouts/_layout.scss [hunk 4]
```diff
@@ -21,6 +21,11 @@ html {
...
 body {
   // Disable in production mode;
-  // @include show-breakpoints;
+  //@include show-breakpoints;
+  
+  font-size: rem(16.1px);
+  line-height: rem(26px);
+  font-weight: 300;
+  @include montserrat();
   text-size-adjust: none;
```
Comment (febdao) [inline web/wp-content/themes/pdjtheme/dist/scss/layouts/_layout.scss]: Pleaser re-oder this block

File: web/wp-content/themes/pdjtheme/dist/styleguide/layouts/layout.twig [hunk 5]
```diff
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     {% block head %}
       <link rel="stylesheet" href="../css/styles.css">
+      <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```
Comment (febdao) [inline web/wp-content/themes/pdjtheme/dist/styleguide/layouts/layout.twig]: Please remove this line, we already import the font via css file

**Pull request merged**